allennlp/common/file_utils.py

```diff
@@ -233,9 +235,42 @@ def cached_path(
     cache_dir = os.path.expanduser(cache_dir)
     os.makedirs(cache_dir, exist_ok=True)
 
+    extraction_path: Optional[str] = None
+
     if not isinstance(url_or_filename, str):
         url_or_filename = str(url_or_filename)
 
+    if urlparse(url_or_filename).scheme == "hf":
+        # Remove the hf:// prefix
+        identifier = url_or_filename[5:]
```

Comment:
```suggestion
    if url_or_filename.startswith("hf://"):
        # Remove the hf:// prefix
        identifier = url_or_filename[5:]
```

equivalent but feels saner?

Reply:
I don't feel strongly about it, but I prefer `startswith()`.